config/servers.yml

```diff
@@ -91,7 +91,6 @@
 #        concurrency:               1
 #        requiredAcks:              1
 #        compressionCodec:          none
-#        autoCommitEnabled:         true
```

Comment:
Is the reason to remove this option to imply `KafkaMessageBus`always does the auto acknowledgement of kafka messages and uses OffsetManager?